studentInfo.py

```diff
@@ -0,0 +1,72 @@
+  fullItems = ['Abyssal Mask','Adaptive Helm','Archangels Staff','Ardent Censer','Unholy Grail','Banner of Command','Banshees Veil','Blade of the Ruined King','Dead Mans Plate','Deaths Dance','Duskblade of Drakathaar','Edge of Night','Essence Reaver','Eye of Equinox','Eye of the Oasis','Eye of the Watcher','Face of the Mountain','Frost Queens Claim','Frozen Heart','Frozen Mallet','Gargoyle Stoneplate','Guardian Angel','Guinsoos Rageblade','Hextech GLP-800','Hextech Gunblade','Hextech Protobelt-01','Iceborn Gauntlet','Infinity Edge','Knights Vow','Liandrys Torment','Lich Bane','Locket of the Iron Solari','Lord Dominiks Regards','Ludens Echo','Manamune','Maw of Malmortius','Mejais Soulstealer','Mercurial Scimitar','Mikaels Crucible','Morellonomicon','Mortal Reminder','Nashors Tooth','Ohmwrecker','Phantom Dancer','Rabadons Deathcap','Randuins Omen','Rapid Firecannon','Ravenous Hydra','Redemption','Righteous Glory','Rod Of Ages','Ruby Sightstone','Runaans Hurricane','Rylais Crystal Scepter','Spirit Visage','Statikk Shiv','Steraks Gage','Sunfire Cape','Talisman of Ascension','The Black Cleaver','The Bloodthirstier','Thornmail','Titanic Hydra','Trinity Force','Void Staff','Warmogs Armor','Wits End','Yomuus Ghostblade','Zekes Convergence','Zhonyas Hourglass','ZZ-Rot Portal','','','','','','','','']
+  def __init__(self):
+    self.assignRandomK()
+    self.assignRandomD()
+    self.assignRandomA()
+    self.randomChampion()
+    self.randomLane()
+    self.randomItem1()
+    self.randomItem2()
+    self.randomItem3()
+    self.randomItem4()
+    self.randomItem5()
+    self.randomItem6()
+
+  def randomLane(self):
+     self.lane = random.choice(self.lanes)
+
+  def randomChampion(self):
+     self.mainChamp = random.choice(self.mainChamps)
+  
+  def assignRandomK(self):
+     self.K = random.randint(0,30)
+  def assignRandomD(self):
```

Comment:
[[PEP 8]](https://www.python.org/dev/peps/pep-0008/)

You should insert a line break here, as there should be a line breaks between methods.